packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1521,9 +1521,9 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
   Offset get _paintOffset {
     switch (_viewportAxis) {
       case Axis.horizontal:
-        return Offset(-offset.pixels, 0.0);
+        return Offset(-offset.pixels!, 0.0);
       case Axis.vertical:
-        return Offset(0.0, -offset.pixels);
+        return Offset(0.0, -offset.pixels!);
```

Comment:
this ! should become unnecessary when you update to tip of tree

Reply:
AFAICT `pixels` is still nullable in master. I don't understand your comment.

Reply:
ViewportOffset.pixels is non-nullable on master as far as I can tell.

Reply:
humm... It looks like my cherry-pick of the widget migration made ViewportOffset.pixels nullable... I will take a look :-/

Reply:
Done